test/functional/fixtures/regression/gh-850/test.js

```diff
@@ -0,0 +1,9 @@
+describe('[Regression](GH-850)', function () {
+    it('Should resume the test if the cursor is moving from reloaded iframe to top window', function () {
+        this.timeout(40000);
```

Comment:
Do you mean it's ok that the tests is not finished in 30 seconds?? You should found out why it happened